jetson/power_logging/README.md

```diff
@@ -47,24 +47,56 @@ OS - Ubuntu 22.04-based root file system
 > Use this exact Docker image to ensure compatibility with `tensorrt==10.1.0` and `torch_tensorrt==2.4.0`.</br>
 > Base image `nvcr.io/nvidia/pytorch:24.06-py3-igpu` might take some time to download on Jetson. (approx. 5 GB in size)
 
-3. Run the container
+3. Run the container to collect power and runtime measurements for the CNN models
 
     ```bash
-    sudo docker run -e DAGSHUB_USER_TOKEN=<dagshub-token> --runtime=nvidia --ipc=host -v $(pwd):/app -d edge-vision-benchmark
+    sudo docker run --runtime=nvidia --ipc=host -v $(pwd):/app -d edge-vision-benchmark
     ```
 
-> [!NOTE]  
-> You can generate a long lived app DagsHub token with no expiry date from your [User Settings](https://dagshub.com/user/settings/tokens).
+    This will start running the [`run_experiment.sh`](./run_experiment.sh) script by default. You can also override by passing your custom experiment script.
 
-This will start running the [`run_experiment.sh`](./run_experiment.sh) script by default. You can also override by passing your custom experiment script. More information on the `run_experiment.sh` can be found in the [data collection on Jetson](../../docs/ExperimentScripts.md#data-collection-script-on-jetson) section.
+    More information on the `run_experiment.sh` can be found in the [data collection on Jetson](../../docs/ExperimentScripts.md#data-collection-script-on-jetson) section.
 
-To follow the logs of the experiment, you can run the following command
+    To follow the logs of the experiment, you can run the following command
 
-```bash
-sudo docker logs -f <container-name>
-```
+    ```bash
+    sudo docker logs -f <container-name>
+    ```
+
+    You can find the name of the docker container using the `sudo docker ps` command.
+
+4. Add DVC credentials to the Jetson as shown in the video below. Run the commands corresponding to the `Add a DagsHub DVC remote` and `Setup credentials` sections on the Jetson.
+
+    <a href="DVC Remote"><img src="./assets/dvc-remote.gif" align="center" height="500" width="500" ></a>
+
+5. Upload benchmark data to DagsHub from Jetson.
+
+    If you are using DVC for **only the first time**, run the following command in the current working directory on the Jetson.
+
+    ```bash
+    dvc init
```

Comment:
This is never required, since people would have cloned a repo with dvc already initialised, right?

Reply:
These are instructions for the developer. I am assuming raw data will be collected by the developers. 

Terminology for clarity

Developer: FuzzyLabs team and someone who wants to run everything end to end right from collecting raw data on the Jetson.

User: Interested in getting training data and running or training models. Others will be running inference using trained models.

Reply:
In the jetson context, it is required for the very first time which is our case as there is no DVC at the moment.

Reply:
If someone is trully running it from scratch, then yeah. However, after we upload the current version of the data, we will have already initialised dvc in this repository; there will be .dvc folder and *.dvc files and all that. 

My expectation for uses:
* FL: we already have access to both gh repo and dagshub repo, so we stick with existing repo and already initialised DVC
* Outside developers that want to train the model: use existing data, run existing pipeline, they have pull access to Dagshub
* Outside developers that want to collect their own data: they fork dagshub repo, change the remote to their repo, and point DVC to their repo (instructions for that can come at a later stage?)

I don't think we should expect users to clear the existing repo, and initialise it again.

Reply:
> I don't think we should expect users to clear the existing repo, and initialise it again.

The above instruction is for developers and not users.

Reply:
Here is an example, if we log in to the Jetson as it is in current state and we have no DVC intialized. How do we communicate to FL developers who are running this for the very first time that they are supposed to run `dvc init` as well?

Reply:
>Which is our case here I think.

Yes it is. All I'm saying that ones this PR and #34 are merged, it's unlikely someone is going to run it entirely from scratch, rather then using our or a forked DVC repo. 

I'm alright with keeping the instructions though, if we add a note that this is only required when the existing DVC repo was cleared for some reason, and the dev starts from scratch

Reply:
FYI: We will add `dvc init` command in model_training folder as well since it is not being tracked right now for training data.

Let me know what note should be added to be more clear and explicit for developers that this should only be done only for the first time of tracking the dataset folders.